Base/usr/share/man/man7/setuid_overview.md

```diff
@@ -0,0 +1,71 @@
+## Name
+
+Overview of real, effective, and saved user and group ids
+
+## Description
+
+Each process runs as user and a group. By default, a process started by a user has its user id set to that user and its group id set to that user's primary group.
+
+Each file has permissions that control if it can be read, written, and executed by its owner, other users in the owner's group, and by others. A process has the same access permissions as its user and its group.
```

Comment:
This, too, works the other way around. It's not that the process has the permissions of its user; those user permissions determine exactly that, what processes running as that UID/GID can do. The user cannot do anything by themselves other than generate some interrupts by moving mouse & typing on the keyboard; in particular they cannot access the files directly. It's always processes that act in behalf of the user.